tools/generate_tables_sql_and_testNames.php

```diff
@@ -99,7 +99,8 @@
     $fp = fopen($filename, "w");
     fwrite($fp, $output);
     $sql = "REPLACE INTO test_names (Test_name, Full_name, Sub_group) "
-         . "VALUES ('$tablename', '$title', 1);";
+         . "SELECT '$tablename', '$title', ID "
+         . "FROM test_subgroups WHERE Subgroup_name='Instruments';\n";
```

Comment:
I don't think there's any guarantee that there's a subgroup named "Instruments" either. We don't have any guarantees for what the subgroups names are other than that there's at least 1 of them, which I think is why it was using a hardcoded 1. This might just be trading it breaking on some environments for breaking on others. I'm not sure what the solution is here.

Two options I can think of are:
1. Prompt for which subgroup it should go into in the instrument manager. (Probably a lot more work.)
2. Use `MIN(ID)` (or `MAX(ID)`) (minimal work and should work in all environments but will sometimes go into the wrong subgroup.. though that's also the case now.)

Reply:
isn't in guaranteed if its part of our schema? https://github.com/aces/Loris/blob/189a43d15dfba0c6584f1264682a6baa0da9b874/SQL/0000-00-00-schema.sql#L249

if a DB veers from that, then they should have an override to ensure there's no breaking changes?

Reply:
No, that's just the default data. They can organize it however they want. Many projects group them differently.

Reply:
But "Instruments" is a database value here, not a part of the schema.

Reply:
yeah the value is inserted as part of the schema (linked above)

Reply:
Hi @driusan, the original issue reported by me was due a misconfiguration (mean the issue don't appear in a properly configured environment @zaliqarosli and myself tested.)

Regarding "Instruments" or "1"  agree is more a "feature" than a "bug fix" but since "Instruments" is inserted "by a default installation" I don't see it worse than "1".